### ISSUE TITLE
[CF Summit 2018 Basel] Add labels / titles to pipelines

### ISSUE THREAD

**Issue by domdom82** [concourse/concourse#2688]
## What challenge are you facing?

This issue was raised in an open space discussion during the CF Summit unconference at Base 2018.

When concourse is used to deploy many CF deployments for different regions, people will usually have one concourse per region. This means for each region they will stare at the same pipeline which will look exactly the same for every region. When opening a browser window and pointing it to the concourse pipeline, other than the URL there is no distinction if that pipeline is for region x or region y.

To work around this problem we are currently abusing job groups to display one group for the entire pipeline.

## What would make this better?

Pipelines should have a label / title field that is displayed on the UI when the pipeline is displayed.

## Are you interested in implementing this yourself?

Not a UI guy, sorry :)

**Comment by jpluscplusm** [comment]
Isn't the pipeline name, displayed top-left in the pipeline view (https://ci.concourse-ci.org/teams/main/pipelines/concourse) the right place to distinguish this? And isn't it (IIRC!) template- or suffix-able?

**Comment by domdom82** [comment]
Yes, that's possible. Though ugly. Let's say you have 10 pipelines and 10 regions where concourse is deployed. What you actually want is this:
```
REGION 1
Pipeline Menu
-- pipeline_A
-- pipeline_B
(...)
```
instead what you get is:
```
Pipeline Menu
-- pipeline_A (REGION 1)
-- pipeline_B (REGION 1)
(...)
```

This can get especially ugly when your pipeline names + region names are too long and get word-wrapped like:
```
Pipeline Menu
-- pipeline_A (REG
                       ION 1)
-- pipeline_B (REG
                       ION 1)
(...)
```

So I guess this is more about a "concourse instance name" title bar than pipelines per se...

**Comment by domdom82** [comment]
@jama-pivotal thanks for pointing out the right issue this is exactly our problem 👍
